shell/list/provisioning.cattle.io.cluster.vue

```diff
@@ -34,68 +35,30 @@ export default {
     }
   },
 
-  async fetch() {
-    this.$initializeFetchData(CAPI.RANCHER_CLUSTER);
-    const hash = {
-      rancherClusters: this.$fetchType(CAPI.RANCHER_CLUSTER),
-      normanClusters:  this.$fetchType(NORMAN.CLUSTER, [], 'rancher'),
-      mgmtClusters:    this.$fetchType(MANAGEMENT.CLUSTER),
-    };
-
-    this.$store.dispatch('rancher/findAll', { type: NORMAN.CLOUD_CREDENTIAL });
-
-    if ( this.$store.getters['management/canList'](SNAPSHOT) ) {
-      hash.etcdSnapshots = this.$fetchType(SNAPSHOT);
-    }
-
-    if ( this.$store.getters['management/canList'](CAPI.MACHINE) ) {
-      hash.capiMachines = this.$fetchType(CAPI.MACHINE);
-    }
-
-    if ( this.$store.getters['management/canList'](MANAGEMENT.NODE) ) {
-      hash.mgmtNodes = this.$fetchType(MANAGEMENT.NODE);
-    }
-
-    if ( this.$store.getters['management/canList'](MANAGEMENT.NODE_POOL) ) {
-      hash.mgmtPools = this.$fetchType(MANAGEMENT.NODE_POOL);
-    }
-
-    if ( this.$store.getters['management/canList'](MANAGEMENT.NODE_TEMPLATE) ) {
-      hash.mgmtTemplates = this.$fetchType(MANAGEMENT.NODE_TEMPLATE);
-    }
-
-    if ( this.$store.getters['management/canList'](CAPI.MACHINE_DEPLOYMENT) ) {
-      hash.machineDeployments = this.$fetchType(CAPI.MACHINE_DEPLOYMENT);
-    }
-
-    // Fetch RKE template revisions so we can show when an updated template is available
-    // This request does not need to be blocking
-    if ( this.$store.getters['management/canList'](MANAGEMENT.RKE_TEMPLATE_REVISION) ) {
-      this.$fetchType(MANAGEMENT.RKE_TEMPLATE_REVISION);
-    }
-
-    const res = await allHash(hash);
-
-    this.mgmtClusters = res.mgmtClusters;
-  },
-
   data() {
```

Comment:
there's an error in console

```
[Vue warn]: Property "loadResources" was accessed during render but is not defined on instance.
```
Looks like loadResources comes from the resource list mixins, however even before these changes i dont think it was exposed?

Reply:
it was exposed by the resourceFetch mixin. It's no longer necessary since it's the PaginatedResourceTable that's doing the fetching.

I've removed it.